src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.Service/Microsoft.PowerToys.Run.Plugin.Service.csproj

```diff
@@ -71,6 +71,7 @@
   
   <ItemGroup>
     <ProjectReference Include="..\..\Wox.Plugin\Wox.Plugin.csproj" />
+    <ProjectReference Include="..\..\Wox.Infrastructure\Wox.Infrastructure.csproj" />
```

Comment:
Something to ponder about. It feels like plugins should be independent and reference to Wox.Infrastructure violates it. It is okay to have reference to a project that contains plugin interface. What do you think? Does it make sense?

Reply:
Oh.. I missed this! Almost all plugin are referencing `Wox.Infrastructure`. Consider also that `ISavable` and `PluginJsonStorage` are in `Wox.Infrastructure`. Could be a refactoring. Should I open an issue?

Reply:
We can open a separate issue for that and discuss the pros and cons there. It is definitely a low-priority one.